[2022-05-26 14:53:11,720] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-05-26T14:53:08.315533+00:00 [queued]>
[2022-05-26 14:53:11,726] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-05-26T14:53:08.315533+00:00 [queued]>
[2022-05-26 14:53:11,726] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 14:53:11,726] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-26 14:53:11,726] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 14:53:11,734] {taskinstance.py:1378} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-05-26 14:53:08.315533+00:00
[2022-05-26 14:53:11,737] {standard_task_runner.py:52} INFO - Started process 532 to run task
[2022-05-26 14:53:11,739] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2022-05-26T14:53:08.315533+00:00', '--job-id', '16', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmprtzk1p_e', '--error-file', '/tmp/tmpy8_rd_n0']
[2022-05-26 14:53:11,739] {standard_task_runner.py:80} INFO - Job 16: Subtask run_after_loop
[2022-05-26 14:53:11,769] {task_command.py:370} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2022-05-26T14:53:08.315533+00:00 [running]> on host b7bcf2b221fa
[2022-05-26 14:53:11,810] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-05-26T14:53:08.315533+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-26T14:53:08.315533+00:00
[2022-05-26 14:53:11,810] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-26 14:53:11,810] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2022-05-26 14:53:11,817] {subprocess.py:85} INFO - Output:
[2022-05-26 14:53:11,818] {subprocess.py:92} INFO - 1
[2022-05-26 14:53:11,818] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-26 14:53:11,835] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220526T145308, start_date=20220526T145311, end_date=20220526T145311
[2022-05-26 14:53:11,870] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-26 14:53:11,889] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
